docs/sources/shared/datasources/tempo-traces-to-profiles.md

```diff
@@ -0,0 +1,34 @@
+---
+headless: true
+labels:
+  products:
+    - enterprise
+    - oss
+---
+
+[//]: # 'This file documents the Traces to profile configure and usage for the Tempo data source.'
+[//]: # 'This shared file is included in these locations:'
+[//]: # '/grafana/docs/sources/datasources/tempo/configure-tempo-data-source.md'
+[//]: # '/website/docs/grafana-cloud/data-configuration/traces/traces-query-editor.md'
+[//]: #
+[//]: # 'If you make changes to this file, verify that the meaning and content are not changed in any place where the file is included.'
+[//]: # 'Any links should be fully qualified and not relative: /docs/grafana/ instead of ../grafana/.'
+
+<!-- # Trace to profiles  -->
+
+{{< docs/experimental product="Trace to profiles" featureFlag="traceToProfiles" >}}
+
+Using Trace to profiles, you can use Grafana’s ability to correlate different signals by adding the functionality to link between traces and profiles.
+
+**Trace to profiles** lets you link your Grafana Pyroscope data source to tracing data.
+When configured, this connection lets you run queries from a trace span into the profile data.
+
+To use tract to profiles, you must have a configured Grafana Pyroscope data source.
+
+1. Select the Pyroscope data source from the **Data source** drop-down.
+1. Optional: Choose any tags that will be used in the query. If left blank. the default values of `service.name` and `service.namespace` are used.
```

Comment:
What are the parameters for the tags? When we say tags, what exactly do we mean?

Reply:
I think this is explained well in the trace to logs [docs](https://grafana.com/docs/grafana/latest/datasources/tempo/configure-tempo-data-source/#configure-a-custom-query).

Reply:
Are the tags for profiles the same as the ones for Trace to logs? The table in the Trace to logs content would be great to include, if it applies.

Reply:
Yes, we can use the same tags table.

Reply:
Added.